docs/setup/mac.md

```diff
@@ -97,6 +97,6 @@ If VS Code doesn't update once it restarts, it might be set under quarantine by
 
 ### Does VS Code run on Mac M1 machines?
 
-Yes, VS Code supports macOS ARM64 builds that can run on Macs with the Apple M1 chip. Currently, only [Insiders](/insiders) macOS ARM64 builds are available.
+Yes, VS Code supports macOS ARM64 builds that can run on Macs with the Apple M1 chip starting with version [1.54](/release-notes/v1_54.md) and [Insiders](/insiders) builds.
```

Comment:
I think we can just drop the second sentence and don't need to mention the release number.

Reply:
Thanks @gregvanl ! There's only one sentence, but I _think_ I caught what you're saying. Check my edits?